ui/portal_translation_tool.js

```diff
@@ -0,0 +1,976 @@
+import ClassUI from "./ClassUI.js";
+
+class PortalTranslationTool extends ClassUI {
+   constructor() {
+      super();
+      this.containerDomID = "domID";
+   }
+
+   ui() {
+      const translationToolBody = {
+         view: "scrollview",
+         css: "lightgray ab_amp",
+         body: {
+            rows: [
+               {
+                  view: "accordion",
+                  id: "qtt_accordion_" + this.containerDomID,
```

Comment:
I think in general we're trying to use string literal syntax, but not a big deal to me.
```js
id: `qtt_accordion_${this.containerDomID}`,
```

Reply:
Okay I switched all items I could find to use literal syntax.